### PR TITLE
Updated the readme

### PR DESCRIPTION
**Pull request by Abhishek-IOT**
Updated the readme for more clearance.

### CODE REVIEW

**Comment by Abhishek-IOT** [comment]
Can anyone help me with

File: README.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ browsers.
 This is accomplished by providing a HTML/JavaScript GUI for entering source
 code and visualization of output, and a server part which evaluates the
 requested source code. Note that this shell is not scalable and it uses
-only one instance on GAE, thus all evaluation requests are queued and it
+only one instance on GAE(Google App Engine),thus all evaluation requests are queued and it
```
Comment (waldyrious) [inline README.rst:14]: IMO it would be better to add the GAE acronym in parenthesis after the first instance of the expression "Google App Engine", in line 8 above.

Comment (Abhishek-IOT) [reply README.rst:14]: Ok sure I will do it @waldyrious (Thanks for your suggestion).

**Comment by Abhishek-IOT** [comment]
My docker-compose up command is taking too much time to start can anyone just help me with the development enviroment to create so that I can add things in the main project also.

File: README.rst [hunk 2]
```diff
@@ -5,13 +5,13 @@ SymPy Online Shell
...
 Online Shell for SymPy (sympy-live) is a simple web application based on
-Google App Engine, which allows to evaluate Python code with SymPy in web
+Google App Engine(GAE), which allows to evaluate Python code with SymPy in web
 browsers.
 
 This is accomplished by providing a HTML/JavaScript GUI for entering source
 code and visualization of output, and a server part which evaluates the
 requested source code. Note that this shell is not scalable and it uses
-only one instance on GAE, thus all evaluation requests are queued and it
+only one instance on GAE,thus all evaluation requests are queued and it
```
Comment (waldyrious) [inline README.rst:14]:
```suggestion
only one instance on GAE, thus all evaluation requests are queued and it
```

File: README.rst [hunk 3]
```diff
@@ -5,13 +5,13 @@ SymPy Online Shell
     :target: https://travis-ci.org/sympy/sympy-live
 
 Online Shell for SymPy (sympy-live) is a simple web application based on
-Google App Engine, which allows to evaluate Python code with SymPy in web
+Google App Engine(GAE), which allows to evaluate Python code with SymPy in web
```
Comment (waldyrious) [inline README.rst:8]:
```suggestion
Google App Engine (GAE), which allows to evaluate Python code with SymPy in web
```

File: templates/shell.html [hunk 4]
```diff
@@ -15,7 +15,7 @@ <h1>SymPy Live</h1>
         <h3>{{ user.nickname }}</h3>
     </div>
     {% else %}
-    <h3><a href="{{ login_url }}">Log In</a></h3>
+
```
Comment (waldyrious) [inline templates/shell.html:18]: Was this removal intentional?...

Comment (Abhishek-IOT) [reply templates/shell.html:18]: No I have mistaken.

File: README.rst (hunk 1 above)
Comment (Abhishek-IOT) [reply README.rst]: @waldyrious Why always my test fail everytime?

Comment (waldyrious) [reply README.rst]: I have no idea, sorry :(

**Comment by aktech** [comment]
@Abhishek-IOT Can you please address comments above and let me know after that?

**Comment by Abhishek-IOT** [comment]
@aktech Sir,I am not able to get you,what I have to do ?

**Comment by Abhishek-IOT** [comment]
Sir...I had updated the readme file and added the documentation link to solve the problem but they were not accepted.
